README.md

```diff
@@ -2,8 +2,9 @@
 
 ## Status
 
-__Work in progress__.
+The package is available in the Maven Central repository: [ais-service-discovery-java](https://search.maven.org/artifact/com.github.peak-ai/ais-service-discovery-java/1.2/jar) 
```

Comment:
```suggestion
The package is available in the Maven Central repository: [ais-service-discovery-java](https://search.maven.org/artifact/com.github.peak-ai/ais-service-discovery-java) 
```
Since version is going to change. We should keep this url static